udfs/index.md

```
@@ -0,0 +1,417 @@
+```
+
+All that's left is to load this UDF
+```python
+from falkordb import FalkorDB
+
+db = FalkorDB()
+g = db.select_graph("G")
+
+# Load UDF
+with open("./traversals.js", "r") as f:
+	content = f.read()
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)